packages/related-keyphrase-suggestions/src/elements/TrendGraph/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+/* Internal dependencies */
+import TrendGraphTable from "./TrendGraphTable";
+
+/**
+ * Renders a SVG area chart.
+ *
+ * @param {array}    data                      Array of objects with X and Y coordinates for the SVG chart points.
```

Comment:
Might be nice to specify exactly what is expected in the object via the type itself.

Edit: seems it is actually just an array of Y coordinates? The index makes the X (month)